README.md

```diff
@@ -19,67 +19,36 @@
 
-### Hosting instructions
-
-Any chair, editor, or other party authorized by CCG to manage recordings and
-minutes can do the following. A scribe-bot will show up in the main #ccg IRC
-channel automatically.
-
-1. Make sure to select "Start Recording" at the beginning of the call and "Stop
-   Recording" when you're done.
-
-1. Select "Start Recording" to start audio recording
-
-1. Post the link to today's agenda in the meeting chat box. The link below leads 
-   to the default "standing" agenda. If a date-specific agenda was sent to the
-   mailing list prior to the meeting, a link to that archived email should be used
-   instead.
-
-   `Agenda: https://github.com/w3c-ccg/traceability-interop/blob/main/AGENDA.md`
-
-1. Address IP note, agenda review, and scribe selection topic
-
-    - Note the topic in the chat box
-      `Topic: IP Note, Agenda Review, Scribe Selection`
-    - Announce the agenda for the meeting. Meetings alternate between focus on the trace-interop and trace-vocab repositories.
-    - Read the standard IP note
-
-      > Anyone can participate in these calls. However, all substantive contributors to any CCG Work Items must be members of the CCG with full IPR agreements signed. Information about and links to the required license agreements can be found in the meeting agenda.
-
-1. Move on to GitHub Issue & PR review
-
-    - Note the next topic in the chat box
-
-      `Topic: GitHub Issue & PR review`
-
-    - Review pull requests in order (least recently updated first)
-
-      For this portion of the meeting, address PRs in the focus repository first, followed by PRs in the other repository.
-
-        - Publish link to sorted PRs in chat box ([trace-interop](https://github.com/w3c-ccg/traceability-interop/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-asc), [trace-vocab](https://github.com/w3c-ccg/traceability-vocab/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-asc))
-
-        - Publish link to each PR being discussed in chat box
-
-    - Review issues in order (least recently updated first)
-
-      For this portion of the meeting, address issues in the focus repository first, followed by issues in the other repository.
+Historical archives for meetings [can be found here](https://github.com/w3c-ccg/meetings).
 
-        - Publish link to sorted issues in chat box ([trace-interop](https://github.com/w3c-ccg/traceability-interop/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-asc), [trace-vocab](https://github.com/w3c-ccg/traceability-vocab/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-asc))
+### Before the Meeting
 
-        - Publish link to each issue being discussed in chat box
+- Open the [W3C-CCG Traceability Agenda Email Draft](https://docs.google.com/document/d/1Se_PIZNhIzZrwVftbYi-Z3oEMXvucQ7jNjxzjMVWCm4/edit) in Google Docs
+- Update all the items highlighted in yellow, in particular adding new agenda items for week starting with item 7.
```

Comment:
It appears that this will change the original doc, and that the highlighted areas may not be highlighted afterward. Maybe there needs to be a step to "clone"/copy the original draft agenda?

Reply:
Changed "Open" to "Duplicate"